.travis.yml

```diff
@@ -0,0 +1,49 @@
+sudo: false
```

Comment:
Can you add the travis stuff in a seperate PR?  It seems weird to have it here?

Reply:
Sure

Reply:
Created PR #5. However it depends on this PR.

Reply:
Why does it depend on this PR?  That seems surprising?